src/core/mesh/qgsmeshdataprovider.cpp

```diff
@@ -212,6 +212,21 @@ int QgsMesh::edgeCount() const
   return edges.size();
 }
 
+void QgsMesh::setVertices(const QgsMeshVertices& _vertices)
+{
+  vertices=_vertices;
+}
+
+void QgsMesh::setFaces(const  QgsMeshFaces & _faces)
+{
+  faces=_faces;
+}
+
+void QgsMesh::setEdges(const  QgsMeshEdges & _edges)
+{
+  edges=_edges;
+}
```

Comment:
loooks like it doesn't follow the astyle rules? have you run `prepare_commit` script?